translator/src/test/java/com/google/devtools/j2objc/J2ObjCTest.java

```diff
@@ -151,6 +158,19 @@ private void makeAssertionsForJavaFiles(
     makeAssertions(exampleH, exampleM, packageInfoM);
   }
 
+  private void makeAssertionsForGlobalCombinedOutputJavaFiles(
```

Comment:
Please use 4 spaces for method and line continuations (we follow the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html)). Line maximum is 100 columns, so the method parameters can be on the first line, with the throws Exception 4 spaces in from the method's 2 space indent.

Reply:
Updated!